test.js

```diff
@@ -1,24 +1,61 @@
 import test from 'ava';
```

Comment:
```suggestion
import {serial as test} from 'ava';
```

and then you can make it `test.serial` => `test`.